### PR TITLE
Add metrics for infiniband sriov

### PR DESCRIPTION
**Pull request by zshi-redhat**

### CODE REVIEW

**Comment by aneeshkp** [comment]
@zshi-redhat , looks good, just minor update to to the docs/metrics.md.
change this ``` They are grouped by network types for any and sriov only. ```  to include ib-sriov and example.

**Comment by zshi-redhat** [comment]
> @zshi-redhat , looks good, just minor update to to the docs/metrics.md.
> change this `They are grouped by network types for any and sriov only.` to include ib-sriov and example.

@aneeshkp thanks for the comments! 
Added ib-sriov in metric doc.

File: docs/metrics.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ All these metrics are prefixed with `network_attachment_definition_`
 | network_attachment_definition_enabled_instance_up     | Whether or not a  k8s.v1.cni.cncf.io/networks annotated pods are running.  | Gauge   |
                                                         
 
-`network_attachment_definition_instances` -  The number of pod with k8s.v1.cni.cncf.io/networks annotation  and types of networks configured via network attachment definition .  They are grouped by various  network types.
+`network_attachment_definition_instances` -  The number of pod with k8s.v1.cni.cncf.io/networks annotation  and types of networks configured via network attachment definition .  They are grouped by various network types.
```
[the remark this answers is not in the document]
Comment (zshi-redhat) [reply docs/metrics.md]: @s1061123 thanks for capturing it!
Fixed.

**Pull request merged**